### PR TITLE
Draft contributing guide

### PR DESCRIPTION
**Pull request by CohenCyril**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  ```
+  tactic.
+  - tactic. (* secondary subgoal 1 *)
+  - tactic. (* secondary subgoal 2 *)
+  tactic. (* third subgoal is the main one *)
+  ```
+
+## Statements of lemmas, theorems and definitions.
+
+- Universal quantifications with dependent variable should appear on the left hand side of the colon, until we reach the first non dependent variable. e.g.
```
Comment (ybertot) [inline CONTRIBUTING.md]: with dependent variable**s**

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+### Term style.
+- Operators are surrounded by space, for example `n*m` should be written `n * m`.
+This particular example can be problematic if matrix.v is imported because then, `M *m N` is matrix product.
+
+### Naming of variables
+- Variable names follow the following conventions.
+  + Hypothesis should not be named `H`, `H'`,... (these collide with
+  subgroup variable conventions) but have meaningful names. For
+  example, an hypothesis `n > 0` should be named `n_gt0`.
+  + Induction Hypothesies are prefixed by `IH` or `ih` (e.g. induction hypothesis on `n` is called `IH`).
```
Comment (ybertot) [inline CONTRIBUTING.md]: Hypotheses

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  + Natural numbers and integers should be names `m`, `n`, `p`, `d`, ...
```
Comment (ybertot) [inline CONTRIBUTING.md]: should be name**d**

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  + Elements of another ring should be named `x`, `y`, `z`, `u`, `v`, `w`, ...
+  + Polynomials are named by lower case letter `p`, `q`, `r` ... (to avoid collision with properties named `P`, `Q`, ...)
```
Comment (ybertot) [inline CONTRIBUTING.md]: should be named

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  + Matrices are named `M`, `N`, `D`, ...
```
Comment (ybertot) [inline CONTRIBUTING.md]: should be named

**Comment by gares** [comment]
Cyril please fix the typos and let's merge this one!

**Pull request merged**